pages/account/how-to/use-2fa.mdx

```diff
@@ -3,98 +3,117 @@ title: How to use Multifactor Authentication (MFA)
 description: Enable and use Two-Factor Authentication (2FA) in Scaleway.
 tags: authentication 2FA two-factor two multifactor security google authenticator authenticator
 dates:
-  validation: 2025-09-03
+  validation: 2025-10-01
   posted: 2022-01-14
 ---
 import Requirements from '@macros/iam/requirements.mdx'
 
+Multifactor authentication provides extra layers of security. In addition to your password (the first factor), other factors (such as TOTP and passkeys) are required to access your account.
 
-You can enable multifactor authentication on your Scaleway account for heightened security. Once enabled, an additional security code generated by your phone is required to access your account.
+You can enable Multifactor Authentication (MFA) on your Scaleway account for enhanced security. Even in the event of a leaked password, your account is protected.
 
-Multifactor authentication provides extra layers of security. In addition to your password (the first factor), other factors (such as an MFA app) are required to access your account. Even in the event of a leaked password, your account is protected.
+Two MFA methods are available:
+    - **One-time password (TOTP)** - method through which you access your account using a unique, time-based validation code generated by an app.
+    - **Passkey** - secure authentication method that eliminates the need to create, manage, or remember passwords.
+
+<Message type="important">
+  If you are an account Owner, you can set up the two different MFA methods at the same time. We recommend you have at least one of the two MFA methods set up at a time, to keep your account safe.
+</Message>
 
 <Requirements />
 
 - A Scaleway account logged into the [console](https://console.scaleway.com)
 
-## How to download an MFA app
+## How to enable TOTP
 
-To use [multifactor authentication](/account/concepts/#multifactor-authentication-mfa) on your account, you first need to download an MFA app onto your smartphone. Once configured, MFA apps display a constantly rotating set of codes to use with your different accounts when you are prompted for one during login. Popular MFA apps include:
+To use TOTP on your account, you first need to download a TOTP app onto your smartphone. Once configured, TOTP apps display a constantly rotating set of codes to use with your different accounts when you are prompted for one during login. Popular TOTP apps include:
```

Comment:
this doesn't have to be an app on you smartphone, it could be a browser extension, desktop app, or event hardware security key (a yubikey can generate totp codes for example)